Jellyfin.Api/Controllers/ConfigurationController.cs

```diff
@@ -87,7 +88,7 @@ public ActionResult<object> GetNamedConfiguration([FromRoute] string? key)
         public async Task<ActionResult> UpdateNamedConfiguration([FromRoute] string? key)
         {
             var configurationType = _configurationManager.GetConfigurationType(key);
-            var configuration = await JsonSerializer.DeserializeAsync(Request.Body, configurationType).ConfigureAwait(false);
+            var configuration = await JsonSerializer.DeserializeAsync(Request.Body, configurationType, JsonDefaults.GetOptions()).ConfigureAwait(false);
```

Comment:
Maybe it's better to keep these as readonly fields instead of re-creating them every time?

Reply:
I changed the function so the options don't get recreated every time. Did you mean something like that?

Reply:
`JsonDefaults` should stay the same, move the field to the controller

Reply:
Is this the way you wanted it?

Reply:
The controller is initialized on request, so moving the call will have no effect.

Reply:
🤔 
Disregard everything I've said

Reply:
So revert it or leave it?